src/frontend/components/Button.js

```diff
@@ -0,0 +1,7 @@
+const React = require('react')
+
+const Button = (props) => {
+  return (
+   <button className={props.className} onClick={props.onClick}> {props.buttonText} </button> 
+ )}
+ export default Button; 
```

Comment:
This button component is not suppose to be in this PR. It has already been captured in the the Button component PR